### PR TITLE
Spanish translation updated.

### PR DESCRIPTION
**Pull request by igor-cali**

### CODE REVIEW

File: app/src/main/res/values-es/strings.xml [hunk 1]
```diff
@@ -77,29 +58,99 @@
...
+    <!-- Errors -->
     <string name="msg_err_parsing_json">Error al analizar JSON.</string>
     <string name="msg_connection_problem">Hay un problema con su conexión a Internet.</string>
     <string name="msg_connection_not_available">Conexión no disponible.</string>
     <string name="msg_city_not_found">No se encuentra la ciudad especificada.</string>
-
+    <string name="msg_too_many_requests">Demasiadas solicitudes. Inténtelo más tarde.</string>
+
+    <!-- Weather icons -->
+    <string name="weather_cloudy" translatable="false">&#xf013;</string>
```
Comment (robinpaulson) [inline app/src/main/res/values-es/strings.xml:125]: do we need these lines? 125-151? as they are not translated, I was under the impression we should leave them out?

**Comment by robinpaulson** [comment]
Otherwise, this is great, I'll approve it all as soon as I figure out if we need that block or not. I honestly don't know the difference of having it in or out.

File: app/src/main/res/values-es/strings.xml (hunk 1 above)
Comment (igor-cali) [reply app/src/main/res/values-es/strings.xml]: Strings removed.

**Comment by robinpaulson** [comment]
excellent, cheers for that

**Pull request merged**